workers/unity/Packages/com.improbable.gdk.core/Collections/BlittableMap.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Improbable.Gdk.Core.Collections
+{
+    public struct BlittableMap<TKey, TValue> : IDictionary<TKey, TValue>
```

Comment:
What was the reason for keeping Map instead of dictionary?

Reply:
Because this represents a schema Map.

Reply:
But it's a blittable version of the Dictionary class. Unless we mean it to be a new IDictionary which I am against.

Reply:
If you want, I can remove the IDictionary, and just straight implement ICollection and still keep the same Add functions etc as Dictionary has?
Same for the List